deployments/scheduler/plugin.yaml

```diff
@@ -96,8 +97,6 @@ spec:
               cpu: 4
               memory: 2Gi
           volumeMounts:
-            - mountPath: /etc/kubernetes
-              name: scheduler-config
             - mountPath: /etc/yunikorn
               name: quotamanager-config
```

Comment:
With the new configuration v2 do we need these?
I could not find any mounting beside the secrets for the admission controller in the current helm charts.

@craigcondit

Reply:
Hi @wilfred-s , you are right and I will remove this in next commit.
After I removed the quatamanager-config, latest yunikorn will use the default configuration here.
<img width="547" alt="image" src="https://user-images.githubusercontent.com/45888688/218907959-7dd7f59a-dad7-4cfd-bff8-91d915d855a2.png">

Reply:
That looks correct, let's see if we have got this all correct and get a review from Craig.